service/frontend/adminHandler_test.go

```diff
@@ -975,10 +900,10 @@ func Test_UpdateGlobalIsolationGroups(t *testing.T) {
 func Test_IsolationGroupsNotEnabled(t *testing.T) {
 	handler := adminHandlerImpl{
 		Resource: &resource.Test{
-			Logger:          loggerimpl.NewNopLogger(),
-			IsolationGroups: nil, // valid state, the isolation-groups feature is not available for all persistence types
-			MetricsClient:   metrics.NewNoopMetricsClient(),
+			Logger:        loggerimpl.NewNopLogger(),
+			MetricsClient: metrics.NewNoopMetricsClient(),
 		},
+		isolationGroups: nil, // valid state, the isolation-groups feature is not available for all persistence types
 	}
 	assert.NotPanics(t, func() {
```

Comment:
it's probably better to check the error type, which implies that it doesn't panic